test/fixtures/module-require-depth/two.js

```diff
@@ -1,11 +0,0 @@
-// Flags: --expose_internals
-'use strict';
-const assert = require('assert');
-const {
-  requireDepth
-} = require('internal/modules/cjs/helpers');
-
```

Comment:
☝️ Test above was likely borked because it restructured the `requireDepth` property.